app/scripts/modules/core/help/helpContents.js

```diff
@@ -123,6 +123,11 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
     'gce.loadBalancer.advancedSettings.unhealthyThreshold': '<p>Configures the number of unhealthy observations before deservicing an instance from the load balancer.</p><p>Default: <b>2</b></p>',
     'gce.loadBalancer.healthCheck': '(Optional) <b>Health Checks</b> use HTTP requests to determine if a VM instance is healthy.',
     'gce.loadBalancer.portRange': '(Optional) Only packets addressed to ports in the specified <b>Port Range</b> will be forwarded. If left empty, all ports are forwarded. Must be a single port number or two port numbers separated by a dash. Each port number must be between 1 and 65535, inclusive. For example: 5000-5999.',
+    'gce.serverGroup.resizeWithAutoscalingPolicy': `
+      Setting the desired instance count for a server group with an autoscaler is not supported by Spinnaker.
+      A true resize operation for regional server groups will fail, and the behavior is unpredictable for zonal server groups:
```

Comment:
I think you can strike this line entirely. Just attach '...is not supported by Spinnaker.' and 'if the desired instance...' with a semicolon.